style.css

```diff
@@ -419,10 +434,13 @@ textarea {
     margin: 27px 142px;
     align-items: center;
     padding: 0;
+    opacity: 0.6;
```

Comment:
The opacity is a little too low. It makes me want to avoid looking a the projects instead of highlighting them. I would rather make it close to normal and bring it out via a big coloured border shadow or something 🤔

Reply:
i changed this to an enlargement transition in the accessibility branch.